src/interfaces.ts

```diff
@@ -267,11 +267,15 @@ export interface IFeatureFlags {
     [key: string]: number | boolean | string;
 }
 
-export interface IExportConfig {
-    title?: string;
+export interface IExportConfigRequest {
     format?: "xlsx" | "csv" | "raw";
     mergeHeaders?: boolean;
     showFilters?: AFM.CompatibilityFilter[];
+    title?: string;
+}
+
+export interface IExportConfig extends IExportConfigRequest {
```

Comment:
does this interface needs to be in gooddata-js, when it is not using it anywhere. data of this type are pushed from AD/KD into UI SDK and between UI SDK and gooddata-js it is always using "old" showFilters prop. If you move this interface into UI SDK, you don't need any mention in gooddata-js changelog because there is no change. The only one logical change is already done in UI SDK and all documentation/description should be there imho. You can have original IExportConfig in gooddata-js, IExtendedExportConfig in AD/KD/UI SDK and it will be backward compatible change, because old UI SDK code using IExportConfig will still work, because 
`IExtendedExportConfig extends IExportConfig`

Reply:
If `IExtendedExportConfig` is in UI SDK, we will need to update the packages in AD, KD and customer's code. From 
`import { IExportConfig } from '@gooddata/gooddata-js';` 
to 
`import { IExtendedExportConfig } from '@gooddata/react-components';`.

Leave it here help to reduce the number of works.

Reply:
I see your point, but UI SDK user doesn't have to know about internal implementation, that we are using gooddata-js functions. Even before the interface should be reexported by UI SDK to hide internals. Now you have the opportunity to fix it. In AD/KD it is irrelevant, we can fix it there with one search&replace